adminSite/server/apiRouter.ts

```diff
@@ -227,7 +227,7 @@ async function saveChart(
 
         if (existingConfig) {
             // Bump chart version, very important for cachebusting
-            newConfig.version = existingConfig.version + 1
+            newConfig.version = existingConfig.version! + 1
```

Comment:
You are right, and this will indeed break with the new code, because version might be undefined.

Do we have an error Lint rule for non-null assertions? I would be in favor of it. And in the rare cases where it's needed, you would have to add a `// eslint-disable-next-line [rule]`

Reply:
`@typescript-eslint/no-non-null-assertion` is the rule. It's currently disabled, but we can certainly enable it.